src/main/resources/api-definition.yaml

```diff
@@ -508,3 +510,26 @@ resourceTypes:
                   body:
                       application/json:
                           schema: Device
+  /{facilityId}/volume:
```

Comment:
I am not sure if this is the correct mapping. If I think correctly, the full path will be /api/{facilityId}/volume. We probably need some base resource like inventoryItem, catalogItem, or even facility:

- /api/inventoryItems/{facilityId}/volume
- /api/catalogItems/{facilityId}/volume
- /api/facilities/{facilityId}/volume